### PR TITLE
Call status update

### PR DESCRIPTION
**Pull request by Deepak-learner**

### CODE REVIEW

**Comment by Deepak-learner** [comment]
I create function updateCallStatus at sdk to communicate with rc-core. PR => https://github.com/RocketChat/Rocket.Chat.js.SDK/pull/142

 updateCallStatus is used at livechat to update the call status PR => https://github.com/RocketChat/Rocket.Chat.Livechat/pull/624

File: app/theme/client/imports/general/base.css [hunk 1]
```diff
@@ -239,7 +239,7 @@ button {
 	.rc-old .main-content-flex {
```
Comment (murtaza98) [inline app/theme/client/imports/general/base.css:239]: Are this changes required?

Comment (Deepak-learner) [reply app/theme/client/imports/general/base.css:239]: No, it's not needed.
